### ISSUE TITLE
Question: BodyBag Retextured

### ISSUE THREAD

**Issue by LaocheXe** [acemod/ACE3#4275]
I would like to re texture the bodybag in ACE, I am in a WWII milsim unit and want to change the badybag texture to white and add some bloodstains - where is the code that calls for bodybagItem_co.png so I can use that as a reference?

**Comment by PabstMirror** [comment]
The bodybag is in APL folder: https://github.com/acemod/ACE3/tree/master/addons/apl

bodybag.p3d references 
https://github.com/acemod/ACE3/raw/master/addons/apl/data/bodybag_co.paa
https://github.com/acemod/ACE3/raw/master/addons/apl/data/bodybag_nohq.paa

If you are interested in making a cloth texture for it, I think we could find a way to either add a different 3d model or use `hiddenSelectionsTextures` on the existing one.

**Comment by jonpas** [comment]
I'll just use this issue to tag as feature request, swift work @jokoho48 !

**Comment by PabstMirror** [comment]
You will have to wait for 3.7 for this, but all you will have to do is throw this in a mission init.sqf:
```
["ACE_bodyBagObject", "init", {
    params ["_object"];
    _object setObjectTexture [0, "#(rgb,8,8,3)color(1,0,1,1)"];
}, true, [], true] call CBA_fnc_addClassEventHandler;
```
and replace `#(rgb,8,8,3)color(1,0,1,1)"` with your texture file.

You can also do this at the config level
```
class ACE_bodyBagObject: MapBoard_altis_F {
    hiddenSelectionsTextures[] = {"your texture"};
};
```

**Comment by bux578** [comment]
Fixed in #4276
